doc/source/navbar.yml

```diff
@@ -1,11 +1,11 @@
 - file: ray-overview/getting-started
-  title: "Getting Started"
+  title: "Get Started"
 - file: ray-overview/use-cases
-  title: "Use Cases"
+  title: "Usecases"
```

Comment:
is "Usecase" really one word?

Reply:
No, it isn't. The design calls for it regardless - I even brought this up with @simran-2797 and confirmed that it needed to be this way, so I think we need to keep it.

Reply:
Just saw the comment above mine, will change it now :sweat_smile: